dashboard/app/models/levels/pythonlab.rb

```diff
@@ -50,6 +53,14 @@ def self.create_from_level_builder(params, level_params)
     )
   end
 
+  def self.start_directions
+    [['None', nil], ['North', 0], ['East', 1], ['South', 2], ['West', 3]]
```

Comment:
Just curious - if a maze is defined, is there a default direction if 'None' is selected?